config-bootstrapper/src/main/resources/configs/config-bootstrapper/entity-service/hypertrace-core/1_entitytypes.conf

```diff
@@ -0,0 +1,109 @@
+version = 1
+name = entitytypes
+commands = [
+  {
+    upgrade : [
+        {
+          type : EntityType,
+          action : DELETE,
+          data : {
+            tenantId = __default
+            filter = {
+            }
+          }
+        }
+    ]
+    rollback : []
+  },
+  {
+    upgrade : [
+      {
+        type : EntityType,
+        action : ADD,
+        data : {
+          entityTypes = [
+            {
+            	"attributeType" : [
+            		{
+            			"name" : "FQN",
```

Comment:
indentation seems to be messed up here.

Reply:
@buchi-busireddy  I copied the complete file from existing location entity services location : https://github.com/hypertrace/entity-service/blob/master/helm/configs/1_entitytypes.conf

I can try fixing existing formatting issues, but not sure if that needs to be done as part of this PR.